controllers/Index.php

```diff
@@ -434,8 +434,20 @@ protected function addPageSyntaxFields($formWidget, $page)
                 unset($fieldConfig['fields']);
             }
 
-            $fieldConfig['cssClass'] = 'secondary-tab ' . array_get($fieldConfig, 'cssClass', '');
-            $formWidget->secondaryTabs['fields']['viewBag[' . $fieldCode . ']'] = $fieldConfig;
+			$placement = (!empty($fieldConfig['placement']) ? $fieldConfig['placement'] : NULL);
```

Comment:
Spaces, not tabs please. I prefer tabs myself too, but everything in October is spaces, so please follow that convention.